### PR TITLE
Cere Turn-key Private Network Milestones 1 and 2

### PR DESCRIPTION
**Pull request by spoluyan**
# Grant Application Checklist

- [x] The [application-template.md](https://github.com/w3f/Open-Grants-Program/blob/master/applications/application-template.md) has been copied, renamed ( "project_name.md") and updated.
- [x] A BTC address for the payment of the milestones is provided inside the application.  
- [x] The software of the project will be released under the Apache license version 2.0 as specified in the [terms and conditions](https://gist.github.com/Noc2/75bc58e8ce9b5d419ff883b0cf2b8c19).
- [x] The total funding amount of the project is below CHF 30k at the time of submission.

### CODE REVIEW

**Comment by Noc2** [comment]
We will take a look at this amendment together with the [milestone evaluation](https://github.com/w3f/Grant-Milestone-Delivery/pull/67).

**Comment by semuelle** [comment]
Is there a reason you deleted the cost and duration of the milestones?

**Review by semuelle — DISMISSED** [review]
Just moves two deliverables to M2 and updates URLs/paths.

File: applications/Cere_Turnkey_Private_Blockchain_Network.md [hunk 1]
```diff
@@ -47,60 +49,52 @@ There will be three primary directories in this repository:
...
 
 * **Total Estimated Duration:** 9 weeks
 * **Total Costs:** 1.8 BTC
 
 ### Milestone 1
 
-* **Estimated Duration:** 5 weeks 
-* **Costs:** 1 BTC
-* **Main Goal:** Basic functionality: Private node, Smart Contract implementation and setup guide.
-
```
Comment (mmagician) [inline applications/Cere_Turnkey_Private_Blockchain_Network.md:69]: Keep these please, but adjust the cost of M1 & M2

File: applications/Cere_Turnkey_Private_Blockchain_Network.md [hunk 2]
```diff
@@ -47,60 +49,52 @@ There will be three primary directories in this repository:
...
+| 2c.  | [Programmatic Asset Transfer](https://github.com/Cerebellum-Network/private-standalone-network-node/blob/dev/ext20/lib.rs#L100) | Smart Contract transfer function allows for the programmatic/automated transfer of tokens from one application/user via smart contract to the other. | Done |
+| 2d.  | [Asset Restrictions](https://github.com/Cerebellum-Network/private-standalone-network-node/blob/dev/docs/asset_restrictions.md) | Support for the locking of assets by time or by issuer permission, support for expirations and potentially invalidations. | Done |
+| 3.  | [Smart Contract Tests](https://github.com/Cerebellum-Network/private-standalone-network-node/blob/dev/ext20/lib.rs#L196) | The Smart Contract implementation will include unit tests, we will be using the off-chain test environment that ink! provides. | Done |
```
Comment (mmagician) [inline applications/Cere_Turnkey_Private_Blockchain_Network.md:77]: It's not necessary to mark deliverables as `Done` here. For this, we have the Grant Milestone Delivery repo, it's enough to submit there. Please remove from here.

File: applications/Cere_Turnkey_Private_Blockchain_Network.md [hunk 3]
```diff
@@ -47,60 +49,52 @@ There will be three primary directories in this repository:
...
 
 ### Milestone 2
 
-* **Estimated Duration:** 4 weeks 
-* **Costs:** 0.8 BTC
```
Comment (mmagician) [inline applications/Cere_Turnkey_Private_Blockchain_Network.md:86]: Adjust

File: applications/Cere_Turnkey_Private_Blockchain_Network.md [hunk 4]
```diff
@@ -2,7 +2,7 @@
 
 * **Project:** Turn-key Private Standalone Blockchain Network
 * **Proposer:** [Cere-Network](https://github.com/Cerebellum-Network)
-* **Payment Address:** 1BbyAGddobdPgNo9c63xsRnwK5hen8pV7F 
+* **Payment Address:** 3A79EuVJQzZPZSYZ4LDMCxqNUy95xJtFng
```
Comment (mmagician) [inline applications/Cere_Turnkey_Private_Blockchain_Network.md:5]: Is there a reason for this change?

**Review by mmagician — CHANGES_REQUESTED** [review]

**Comment by mmagician** [comment]
@spoluyan Since you're moving two of the deliverables to M2, please adjust the cost accordingly (swapping M1 & M2 costs). 

Also, is there a reason why you're changing the Payment Address? Unless you have a very strong reason for this, we suggest keeping the original one to avoid potential conflicts. As you'll have read in the [application template](https://github.com/w3f/Open-Grants-Program/blob/master/applications/application-template.md):
> The above combination of your GitHub account submitting the application and payment address will be your unique identifier during the program. Please keep them safe.

**Comment by mmagician** [comment]
@spoluyan Any updates here?

File: applications/Cere_Turnkey_Private_Blockchain_Network.md (hunk 4 above)
Comment (freddiemac) [reply applications/Cere_Turnkey_Private_Blockchain_Network.md:5]: Hi @mmagician, it was just a preference to use a cold wallet storage vs a custodial one. Let us know if this is an issue.

**Comment by semuelle** [comment]
@spoluyan Could you integrate the changes into this pull request, please? And do I understand correctly that you decided not to move the `Fee Abstraction` and `Article` deliverables to M2?

**Comment by freddiemac** [comment]
> @spoluyan Could you integrate the changes into this pull request, please? And do I understand correctly that you decided not to move the `Fee Abstraction` and `Article` deliverables to M2?

Hi @semuelle, thank you for the input. We are ready to submit milestone 2 also (probably tomorrow), all of the deliverables will end up being the same (even if 1 of them was shifted from M1 to M2). We can swap the fees and shift the deliverable in the doc per your request during this final submission as you'd like. 

Overall our team is proud of the contributions we've made in this grant, we think it's a real asset to the w3f community. Look forward to your feedback to the M2 submission as well.

File: applications/Cere_Turnkey_Private_Blockchain_Network.md (hunk 1 above)
Comment (spoluyan) [reply applications/Cere_Turnkey_Private_Blockchain_Network.md]: Thanks for your comment, @mmagician .
Updated.

File: applications/Cere_Turnkey_Private_Blockchain_Network.md (hunk 2 above)
Comment (spoluyan) [reply applications/Cere_Turnkey_Private_Blockchain_Network.md]: Resolved.

**Review by Noc2 — APPROVED** [review]

**Review by mmagician — APPROVED** [review]

**Review by RouvenP — APPROVED** [review]

**Pull request merged**